core/src/com/mygdx/game/client/screen/ChooseUnitFieldDialog.java

```diff
@@ -0,0 +1,58 @@
+package com.mygdx.game.client.screen;
+
+import com.badlogic.ashley.core.ComponentMapper;
+import com.badlogic.ashley.core.Entity;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Dialog;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.mygdx.game.client.component.FieldComponent;
+import com.mygdx.game.client.component.UnitComponent;
+import lombok.extern.java.Log;
+
+import java.util.logging.Level;
+
+@Log
+public class ChooseUnitFieldDialog {
+
+  private static final ComponentMapper<FieldComponent> fieldComponentMapper = ComponentMapper.getFor(FieldComponent.class);
+  private static final ComponentMapper<UnitComponent> unitComponentMapper = ComponentMapper.getFor(UnitComponent.class);
+  private static final Skin uiSkin = new Skin(Gdx.files.internal("dialog/custom/uiskin.json"));
```

Comment:
tak nie, bo to jest ładowane synchroniczne z dysku, to trzeba zrobić przez GameScreenAssets, dodanie metody getSkin() [bez parametrów] i ładowania skina w metodzie gdzie było ładowanie (co będzie troochę redundantne bo ten skin był już wcześniej załadowany, ale z tego co kojarzę to libgdx to zauważy i od razu oznaczy go jako załadowanego)